my-project/src/App.vue

```diff
@@ -36,12 +30,8 @@ export default {
     }
   },
   methods: {
-    next: function() {
-      this.listIndex++;
-      if (this.listIndex > this.compList.length - 1) this.listIndex = 0;  // 暫定処理
-    },
-    back: function() {
-      this.listIndex--;
+    changeComp: function(e) {
```

Comment:
以下のような記法もありますので、覚えておきましょう。
```suggestion
    changeComp (e) {
```